pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -253,6 +253,9 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 				return nil, fmt.Errorf("conversion of id %s to int failed", fields[len(fields)-1])
 			}
 			conn.ID = uint32(val)
+		case strings.Contains(fs, "protoinfo"):
+			fields := strings.Split(fs, "=")
+			conn.TCPState = fields[2]
```

Comment:
I don't like this. I'd rather do some additional parsing and make it clear we look for `state_orig`. Or if you are confident that it will always come first, at least add a comment.

Reply:
Updated with clearer parsing. Thanks!